party_time/party_time.lua

```diff
@@ -197,15 +268,16 @@ local default_handlers = {
                         return
                     end
                     coroutine.sleep_frame()
-                until(#treasure.pool == 0)
+                --TODO: use treasure pool count once it's implmented.
+                until(true)--#treasure.pool == 0)
```

Comment:
Just ftr, I do not plan to be implement it that way. `#treasure.pool` will always be `10` and I think that makes sense. I would loop through it instead, but not sure if I want a function for that in the treasure lib. I will consider it.